sdk/core/core/inc/az_credentials.h

```diff
@@ -104,8 +104,9 @@ typedef struct
  * @param tenant_id an Azure tenant ID
  * @param client_id an Azure client ID
  * @param client_secret an Azure client secret
- * @return AZ_OK = Successfull initialization <br>
- * Other value = Initialization failed
+ * @return An #az_result value indicating the result of the operation:
```

Comment:
I think we should be using `@retval` across all our docs. For this PR, it is fine (unless you want to solve this ultimately right now in this PR). But we should not forget this, let's discuss switching to `@retval` on our meetings.

Reply:
@antkmsft - please file an issue and let's address it across the board if there is a strong desire. I am fine with what we have currently, but also like the mixed approach:
https://github.com/Azure/azure-sdk-for-c/pull/622#discussion_r422412165